### ISSUE TITLE
Cannot read property 'css' of null

### ISSUE THREAD

**Issue by col-panic** [Requarks/wiki#2499]
**Describe the bug**
After upgrade the page often fails with _Cannot read property 'css' of null_ showing. A simple reload 
shows the correct page.

**To Reproduce**
1. Upgrade to newest version with existing page
2. Every page shows _Cannot read property 'css' of null_
3. Clear browser cache ...
4. Some pages show _Cannot read property 'css' of null_ -> reloading the page in the browser solves the problem

**Expected behavior**
Every page should load without problems

**Screenshots**

![grafik](https://user-images.githubusercontent.com/1679857/94230611-a31fa900-ff02-11ea-83d3-a6d9c6d817c8.png)

**Host Info (please complete the following information):**
 - OS: Debian
 - Wiki.js version: 2.5.144
 - Database engine: postgres

**Comment by sutusa** [comment]
I started seeing this after my upgrade as well.

Anytime I navigate to a page for the first time, I get the same above error page (`Cannot read property 'css' of null`).  If I refresh that page, the page will load cleanly/successfully and will stay that way, even if I navigate away from that page and back.

But, if I ever bounce the wiki server/process, the issue will reappear again for the first load of markdown page.  This only happens for the pages that I have created, not for system/admin pages.

I'm reverting to my pre-upgrade backup for now, but this was my setup:
* Docker images via `docker-compose`.
* Backend DB is a MariaDB container.

My upgrade path:
* From Wiki.js version (per admin dashboard): `Wiki.js 2.4.105`
* To Wiki.js version (per admin dashboard): `Wiki.js 2.5.144`

Docker Images via `docker images` command: 
```
REPOSITORY                          TAG                 IMAGE ID            CREATED             SIZE
docker.io/requarks/wiki             2.4                 342b0baab111        3 months ago        350 MB
docker.io/requarks/wiki             2.5                 f74c645c97ee        2 weeks ago         436 MB
```

Hope this helps.  And Thank you!
